src/main/java/ToDoTask.java

```diff
@@ -0,0 +1,12 @@
+public class ToDoTask extends Task {
+
+    public ToDoTask(String description) {
+        super(description);
+    }
+
+    @Override
+    public String toString()
+    {
+        return String.format("[T][%s] %s", getStatusIcon(), description);
```

Comment:
Maybe the type of the event represented by the "T" here should be static since it's a constant?